hs_file_types/urls.py

```diff
@@ -57,6 +62,11 @@
         views.update_dataset_name,
         name="update_filetype_datatset_name"),
 
+    url(r'^_internal/(?P<file_type_id>[0-9]+)/(?P<coverage_type>[A-z]+)/'
+        r'update-coverage-fileset/$',
```

Comment:
should this and the name be aggregation rather than fileset?  The endpoint can update any aggregation coverage metadata.

Reply:
This endpoint is only for updating fileset coverage from coverage data in the contained (nested) aggregations. Other aggregation types don't support nested aggregation.